src/main/kotlin/com/atlassian/performance/tools/report/api/judge/LatencyImpact.kt

```diff
@@ -0,0 +1,11 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import java.time.Duration
+
+class LatencyImpact(
```

Comment:
It's not a well defined model. It will most likely evolve, so let's give it a builder.

- [ ] `LatencyImpact.Builder`